net/dns/quad100_test.go

```diff
@@ -0,0 +1,110 @@
+package dns
+
+import (
+	"context"
+	dns "golang.org/x/net/dns/dnsmessage"
+	"net"
+	"tailscale.com/net/tsdial"
+	"tailscale.com/util/dnsname"
+	"testing"
+)
+
+func TestQuad100Conn(t *testing.T) {
+	f := fakeOSConfigurator{
+		SplitDNS: true,
+		BaseConfig: OSConfig{
+			Nameservers:   mustIPs("8.8.8.8"),
+			SearchDomains: fqdns("coffee.shop"),
+		},
+	}
+	m := NewManager(t.Logf, &f, nil, new(tsdial.Dialer), nil, nil)
+	m.resolver.TestOnlySetHook(f.SetResolver)
+	m.Set(Config{
+		Hosts: hosts(
+			"dave.ts.com.", "1.2.3.4",
```

Comment:
Recommend using either `example.com` or `ts.net` here. `ts.com` is not associated with Tailscale, that is someone else.

Reply:
updated, thanks!